compiler/src/dotty/tools/dotc/printing/DecompilerPrinter.scala

```diff
@@ -51,4 +51,16 @@ class DecompilerPrinter(_ctx: Context) extends RefinedPrinter(_ctx) {
       }
     }
   }
+
+  override protected def toTextCore[T >: Untyped](tree: Tree[T]): Text = {
```

Comment:
Why is not part of `RefinedPrinter`?

Reply:
No, we do not want this representation when debugging.